setup.py

```diff
@@ -32,7 +32,7 @@
         'six',
         'oauthlib',
         'requests_oauthlib'
-    ] + (['kerberos-sspi'] if "win" in sys.platform else ['kerberos']),
+    ] + (['kerberos-sspi'] if "win32" in sys.platform else ['kerberos']),
```

Comment:
please use marker and not rely on this if statement that only works on the machine that builds

Reply:
Forgiveness, but what do you mean by a marker?

Reply:
Ok. I guess you meant it https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies